extensions/amp-script/0.1/amp-script.js

```diff
@@ -641,7 +641,7 @@ export class SanitizerImpl {
     // TODO(choumx): Call mutatedAttributesCallback() on AMP elements e.g.
     // so an amp-img can update its child img when [src] is changed.
 
-    const tag = node.nodeName.toLowerCase();
```

Comment:
These were intentionally `nodeName` because DOMPurify does a DOM clobbering check for it: https://github.com/cure53/DOMPurify/blob/ac5c29cceb2753c15bdd03aabf91ac5d621b070b/src/purify.js#L580